src/ptf/mask.py

```diff
@@ -84,14 +90,14 @@ def __str__(self):
 
 
 def utest():
```

Comment:
This test can be transferred to a separate module (test), it is not needed here. Instead, you can add a short comment to the class